portfolio/src/main/webapp/style.css

```diff
@@ -83,12 +84,15 @@ a {
   background: white;
   border-radius: 0.5rem;
   border: 1px black solid;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
[use `0` instead of `0px`, if possible](https://google.github.io/styleguide/htmlcssguide.html#0_and_Units)

Reply:
Updated with commit 95c7166